Header Header
float64 Latitude
float64 Longitude
float32 Altitude
float32 HDOP
float64 UTM_easting
float64 UTM_northing
string UTC
int8 Zone
string Letter
string raw_data